digitalocean/Manager.py

```diff
@@ -287,4 +287,7 @@ def get_volume(self, volume_id):
         return Volume.get_object(api_token=self.token, volume_id=volume_id)
 
     def __str__(self):
-        return "%s" % (self.token)
+        return "<Manager: %s>" % (self.token)
```

Comment:
I strongly suggest to not return the token at this point. Just as an idea to prevent "token" stolen by mistake